<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RaportinMuokkaus</class>
 <widget class="QWidget" name="RaportinMuokkaus">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>684</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>FreeSans</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Muokkaa raporttia </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="raporttiCombo"/>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="nimikeNappi">
       <property name="text">
        <string>Nimike</string>
       </property>
       <property name="icon">
        <iconset resource="../../pic/pic.qrc">
         <normaloff>:/pic/tag.png</normaloff>:/pic/tag.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="view">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="lisaariviNappi">
       <property name="text">
        <string>Lisää rivi</string>
       </property>
       <property name="icon">
        <iconset resource="../../pic/pic.qrc">
         <normaloff>:/pic/lisaa.png</normaloff>:/pic/lisaa.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="muokkaaNappi">
       <property name="text">
        <string>Muokkaa</string>
       </property>
       <property name="icon">
        <iconset resource="../../pic/pic.qrc">
         <normaloff>:/pic/muokkaa.png</normaloff>:/pic/muokkaa.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="poistaNappi">
       <property name="text">
        <string>Poista rivi</string>
       </property>
       <property name="icon">
        <iconset resource="../../pic/pic.qrc">
         <normaloff>:/pic/poista.png</normaloff>:/pic/poista.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../pic/pic.qrc"/>
 </resources>
 <connections/>
</ui>
